emeritus-tech/landing-pages-libraries
# [CPT-748] Adjust Wharton faculty slider responsiveness.

🎟 [**Jira Ticket**](https://emeritus.atlassian.net/browseCPT-748)



### 🚢 Deployment Instructions

* N/A

### ✅ QA instructions

* Open any Wharton landing page, and scroll down until the Faculty slider section:
  * Slider should look as before/expected
  * Slider should work as expected: Showing 3 facultative persons in desktop, and only 1 in mobile, sliding 1 at a time in both cases

* Check also that the Testimonial Slider functionality was not affected by this.
  
### ⚠️ Risks

* Medium

### 🖼 Previews

* Desktop:
![Screenshot 2020-07-01 13 20 24](https://user-images.githubusercontent.com/251192/86270991-89d8bc00-bba2-11ea-99ce-f22215f75be4.png)

* Mobile:
![Screen Shot 2020-07-01 at 1 27 13 PM](https://user-images.githubusercontent.com/251192/86271051-9f4de600-bba2-11ea-96fc-11e8c6572d86.png)


### 📝 Notes

* `goto-last` and `goto-first` arrows have been removed, since given the complexity on maintaining those across desktop and mobile while changing the showed slides. Right arrow will disappear when you've reached the end and left arrow will appear once you swipe after 1st one.

* Also fix some heading minor cosmetics that the standardization brought  in Wharton and INSEAD.
